utils/activityLogs/workerLog.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * utils/activityLogs/workerLog.js
+ */
+const winston = require('winston');
+
+/**
+ * Worker log function which serves as constructor with different fields
+ * initialized
+ * @param {String} jobType - Type of worker job
+ */
+function WorkerLog(jobType) {
+  this.jobType = jobType;
+  this.activity = 'worker'; // default value
+  this.user = null;
+  this.token = null;
+  this.ipAddress = null;
+  this.totalTime = null;
+  this.queueTime = null;
+  this.workTime = null;
+  this.dbTime = null;
+  this.recordCount = null;
+  this.errorCount = null;
+
+  // Function to convert worker log object to String format
+  this.toLogString = function toLogString() {
```

Comment:
@iamigo Tried a couple of things, but then after discussion with @shriramshankar, we realized we cannot test the output because the job cannot reach to completion in test mode because there is no worker process.